From 39fae6e206bccca41e8e5c7ecf5cc1b5e3e19e92 Mon Sep 17 00:00:00 2001
From: Eli Collins <eli@apache.org>
Date: Wed, 11 Jul 2012 17:06:41 +0000
Subject: [PATCH 1296/1344] HADOOP-8586. Fixup a bunch of SPNEGO misspellings. Contributed by Eli Collins

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/branches/branch-1@1360298 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit f6ea08def8d5c09719f0043f8f243d7bc818fabb)

Author: Eli Collins
Ref: CDH-4806
---
 example-confs/conf.secure/hdfs-site.xml            |    2 +-
 .../security/AuthenticationFilterInitializer.java  |    2 +-
 .../content/xdocs/HttpAuthentication.xml           |    2 +-
 .../src/documentation/content/xdocs/webhdfs.xml    |    2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/example-confs/conf.secure/hdfs-site.xml b/example-confs/conf.secure/hdfs-site.xml
index b23a4f6..8008d93 100644
--- a/example-confs/conf.secure/hdfs-site.xml
+++ b/example-confs/conf.secure/hdfs-site.xml
@@ -167,7 +167,7 @@
       The HTTP Kerberos principal used by Hadoop-Auth in the HTTP endpoint.
 
       The HTTP Kerberos principal MUST start with 'HTTP/' per Kerberos
-      HTTP SPENGO specification.
+      HTTP SPNEGO specification.
     </description>
   </property>
 
diff --git a/src/core/org/apache/hadoop/security/AuthenticationFilterInitializer.java b/src/core/org/apache/hadoop/security/AuthenticationFilterInitializer.java
index 99572bc..05ccfae 100644
--- a/src/core/org/apache/hadoop/security/AuthenticationFilterInitializer.java
+++ b/src/core/org/apache/hadoop/security/AuthenticationFilterInitializer.java
@@ -32,7 +32,7 @@ import java.util.Map;
 
 /**
  * Initializes Alfredo AuthenticationFilter which provides support for
- * Kerberos HTTP SPENGO authentication.
+ * Kerberos HTTP SPNEGO authentication.
  * <p/>
  * It enables anonymous access, simple/speudo and Kerberos HTTP SPNEGO
  * authentication  for Hadoop JobTracker, NameNode, DataNodes and
diff --git a/src/docs/src/documentation/content/xdocs/HttpAuthentication.xml b/src/docs/src/documentation/content/xdocs/HttpAuthentication.xml
index 3b6a15f..46593a5 100644
--- a/src/docs/src/documentation/content/xdocs/HttpAuthentication.xml
+++ b/src/docs/src/documentation/content/xdocs/HttpAuthentication.xml
@@ -110,7 +110,7 @@
 
       <p><code>hadoop.http.authentication.kerberos.principal</code>: Indicates the Kerberos 
       principal to be used for HTTP endpoint when using 'kerberos' authentication.
-      The principal short name must be <code>HTTP</code> per Kerberos HTTP SPENGO specification.
+      The principal short name must be <code>HTTP</code> per Kerberos HTTP SPNEGO specification.
       The default value is <code>HTTP/localhost@$LOCALHOST</code>.
       </p>
 
diff --git a/src/docs/src/documentation/content/xdocs/webhdfs.xml b/src/docs/src/documentation/content/xdocs/webhdfs.xml
index 112032a..c2bffcd 100644
--- a/src/docs/src/documentation/content/xdocs/webhdfs.xml
+++ b/src/docs/src/documentation/content/xdocs/webhdfs.xml
@@ -152,7 +152,7 @@
 <tr><td><code>dfs.web.authentication.kerberos.principal</code></td>
 <td>The HTTP Kerberos principal used by Hadoop-Auth in the HTTP endpoint.
     The HTTP Kerberos principal MUST start with 'HTTP/' per Kerberos
-    HTTP SPENGO specification.
+    HTTP SPNEGO specification.
 </td></tr>
 <tr><td><code>dfs.web.authentication.kerberos.keytab</code></td>
 <td>The Kerberos keytab file with the credentials for the
-- 
1.7.0.4

